noxfile.py

```diff
@@ -0,0 +1,40 @@
+import nox
+
+package = "dbt-hive"
+python_versions = ["3.7", "3.8", "3.9", "3.10", "3.11"]
+dbt_versions = ["1.5.0"]
```

Comment:
Why do we need to hard-code the dbt-version? This will become one more place to update the dbt-version going forward!

Reply:
Removed the dependency of dbt version.